Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM registry.access.redhat.com/ubi9/ubi-micro@sha256:630cf7bdef807f048cadfe7180d6c27eb3aaa99323ffc3628811da230ed3322a
```

Comment:
Add this version to the common configuration PR so we standardize on the version. Could you also add the version string  in a comment so that we know what the sha is referring to.

Reply:
Yes this line was deleted so we are using an environment variable to set the base image from the common configuration